great_expectations/rule_based_profiler/data_assistant/data_assistant_runner.py

```diff
@@ -99,6 +100,7 @@ def run(
 
             Args:
                 batch_request: Explicit batch_request used to supply data at runtime
+                exact_estimation: Global directive for applicable "Rule" objects to use "exact" numeric range estimator.
```

Comment:
I think there would be a benefit to going into a bit more detail here, since it's the public API. 

* What does it mean to do an `exact_estimation`?
* If we set it to `False`, then what 'non-exact' estimation happens with the `ParameterBuilder`?

Reply:
@Shinnnyshinshin Good points.  I will add and re-commit once `develop` is clear and ask you to re-review.  Thank you!

Reply:
@Shinnnyshinshin This was done; hence marking as resolved.  Thanks.